Fix namespace conflicts and snap compatibility in deployment pipeline

motangpuar opened the pull request:
- Change `helm install` to `helm upgrade --install`, so these scripts
    won't stop upon finding that the namespace exists
- Change yq being installed, since snap won't run on containered version
  of ubuntu (Test on K8S jenkins->k8s pipeline)
- Parameterized env.REPO on jenkins manual-start-pipeline (Just in case I want to
  test my own fork instead of yours)
    
Tried to request for code review through gerrit, but I got 403. 
My gerrit account name is `ymarselino` (m11202819@gapps.ntust.edu.tw)  affiliated with OSC Lab Taiwan (NTUST)

![image](https://github.com/user-attachments/assets/72cfe9ac-eef0-42bc-b109-322a6703d488)

JohnKeeneyEST commented:
Hi motangpuar
Unfortunately, this Github version of IT/DEP is a read-only mirror of the repo maintained in O-RAN SC Gerrit https://gerrit.o-ran-sc.org/r/q/project:it/dep 
You will need to create a gerrit review there. 
To push your review you may first need set up an Linux Foundation ID and agree to the O-RAN SC contributor agreement. 
See https://lf-o-ran-sc.atlassian.net/wiki/spaces/ORAN/pages/14386333/Project+Developer+Wiki 

meanwhile, we will have a look at your suggestions. Thanks.

motangpuar commented:
Thank you @JohnKeeneyEST I'll try to push the review on gerrit

motangpuar committed:
UPDATES:
- Replace `helm install` to `helm upgrade --install` (proceed when ns
  exist)
- Install `yq` by download the binary instead of snap (no snap in
  container)
- Jenkins pipeline env.REPO

Change-Id: I346617ee9f593d16a9afcd5bbce397005899ee30
Signed-off-by: ymarselino <m11202819@gapps.ntust.edu.tw>
